test/unit/acs_device_info.test.js

```diff
@@ -771,6 +771,11 @@ describe('ACS Device Info Tests', () => {
         let reportOnuDevicesSpy =
           jest.spyOn(acsDeviceInfoController, 'reportOnuDevices');
 
+        // Mock the device creation on database
+        jest.spyOn(DeviceModel, 'create').mockImplementationOnce(
+          (_, __) => Promise.resolve(_),
+        );
```

Comment:
O primeiro `_` poderia se chamar `device` ou algo do tipo. Evitaria de ter que passar um `_` dentro do `Promise.resolve` para algo mais intuitivo e o `__` poderia ficar apenas como `_` já que não está usando ele

Reply:
Discordo de leve, o propósito do mock é retornar o primeiro argumento, não importa o que seja passado pra ele - pode ser ou um device ou um array de devices, a depender do contexto. Pro `createRegistry` é sempre um array, mas isso pouco importa, porque a gente só está emulando o comportamento do `create` de sempre retornar o que foi criado na base.

Sobre omitir o segundo argumento, acho ruim porque o código depende de passar o segundo argumento, por mais que para o mock seja irrelevante e que o Javascript permita esse tipo de maracutaia. Até porque um teste mais completo deveria validar se o segundo argumento (não) veio com uma sessão válida.

O que pode fazer pra evitar repetir isso ao longo do código é criar uma função do tipo:
```javascript
const mockDeviceCreation = function() {
  jest.spyOn(DeviceModel, 'create').mockImplementationOnce(
    (_, __) => Promise.resolve(_),
  );
}
```
E então trocar o código acima por:
```suggestion
        mockDeviceCreation();
```

Reply:
Pode ser, a gente coloca `arg1` e `arg2`, rsrs

Reply:
show, vou tentar substituir todos os casos então kkkkk